### PR TITLE
UI: message consistency, brevity, usability

### PR DESCRIPTION
**Pull request by justinmk3**
- Show "View Logs" button instead of only mentioning it.
- Show "View Docs" button instead of "Yes" and "No" + verbose message.
- Avoid verbose wording.
- Remove some localization strings from package.nls.json, it is only  needed for messages declared in package.json.
- Don't log "docker ps" invocation.

Screen Shot 2021-08-12 at 5 28 08 PM
Screen Shot 2021-08-12 at 5 27 35 PM

related https://github.com/aws/aws-toolkit-vscode/issues/531

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/awsexplorer/defaultRegion.ts [hunk 1]
```diff
@@ -95,8 +95,9 @@ export async function checkExplorerForDefaultRegion(
             ignoreFocusOut: true,
             title: localize(
                 'AWS.message.prompt.defaultRegionHidden',
-                "This profile's default region ({0}) is currently hidden. Would you like to show it in the Explorer?",
-                profileRegion
+                'Default region "{0}" of profile "{1}" is currently hidden. Show it in the Explorer?',
```
Comment (awschristou) [inline src/awsexplorer/defaultRegion.ts:98]: Consider: Disambiguate (VS Code) Explorer from AWS Explorer by changing "Explorer" to "AWS Explorer"

File: src/shared/utilities/messages.ts [hunk 2]
```diff
@@ -45,18 +45,41 @@ export function makeFailedWriteMessage(filename: string): string {
...
- * @returns	A promise that resolves when the button is clicked or the error is dismissed.
+ * @param message  Message text
+ * @param window  Window
+ * @param kind  Kind of message to show
+ * @param extraItems  Extra buttons shown _before_ the "View Logs" button
+ * @returns	Promise that resolves when a button is clicked or the message is
+ * dismissed, and returns the selected button text.
  */
-export async function showErrorWithLogs(message: string, window: Window = ext.window): Promise<void> {
+export async function showErrorWithLogs(
```
Comment (JadenSimon) [inline src/shared/utilities/messages.ts:55]: Nit: All the optional args should probably be put into a new 'options' arg

**Review by JadenSimon — APPROVED** [review]

File: src/shared/utilities/messages.ts (hunk 2 above)
Comment (justinmk3) [reply src/shared/utilities/messages.ts:55]: Will consider this in a followup PR, where I will also rename `showErrorWithLogs()` .

**Pull request merged**